components/21-atoms/input/_input.scss

```diff
@@ -25,6 +25,11 @@ input[type="password"] {
   }
 }
 
+input[type=date]::-webkit-inner-spin-button { // sass-lint:disable-line no-vendor-prefixes
+  margin-left: .5rem;
```

Comment:
```suggestion
```

Seems kind of weird to apply a margin, then hide it entirely... ? 🤔

Reply:
It does a bit yeah...
but somehow: it does apply the margin and it avoids making another selector to move the 'clear' button to the right.